.github/scripts/test_trymerge.py

```diff
@@ -154,6 +159,14 @@ def test_match_rules(self, mocked_gql: Any, mocked_rmr: Any) -> None:
         repo = DummyGitRepo()
         self.assertTrue(find_matching_merge_rule(pr, repo) is not None)
 
+    @mock.patch('trymerge.gh_graphql', side_effect=mocked_gh_graphql)
+    @mock.patch('trymerge.read_merge_rules', side_effect=mocked_read_merge_rules_raise)
+    def test_read_merge_rules_fails(self, mocked_gql: Any, mocked_rmr: Any) -> None:
+        "Tests that PR fails to read the merge rules"
+        pr = GitHubPR("pytorch", "pytorch", 77700)
+        repo = DummyGitRepo()
+        self.assertRaises(RuntimeError, lambda: find_matching_merge_rule(pr, repo))
```

Comment:
Please note, that this would have been true even before your changes. Perhaps you want to change this one https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex